backend/api.py

```diff
@@ -0,0 +1,93 @@
+from flask import Flask, request, jsonify
+from flask_cors import CORS
+import os
+from copy import copy
+from backend.position import Position
+from backend.piece import Piece
+from backend.board import Board
+from backend.turn import Turn
+template_folder_path = os.path.abspath('./frontend/src')
+app = Flask(__name__, template_folder=template_folder_path)
+CORS(app)
+
+
+board = Board()
+turn = Turn(player="red", eat_turn=False)
+output_json = {}
+check = False
+rows = {}
+@app.route("/", methods=['GET'])
+def board_output():
+
+    for row in range(8):
+        rows["row"+str(row)] = board.get_line_row(row)
+    output_json["board"] = rows
+    output_json["turn"] = turn.get_player()
+    return jsonify(output_json)
+
+
+@app.route('/restart', methods=['GET'])
+def restart():
+    board.reset()
+    turn.set_player("red")
+    return "OK"
+
+
+@app.route('/pick', methods=['POST'])
+def post():
+    requested_data = request.get_json()
+    data = requested_data["data"]
+    if data != "":
+        click_piece = board.get_piece_by_position(data["position"])
+        picked_piece = board.get_picked_piece()
+        pieces_can_eat = board.get_eating_pieces(turn.get_player())
+        can_piece_eat = click_piece.get_position() in pieces_can_eat
+        if click_piece.get_color() == "*" and bool(picked_piece):
+            moving_turn(picked_piece, click_piece)
+        elif click_piece.get_color() == turn.get_player()\
+                and(can_piece_eat or pieces_can_eat == [])\
+                and not turn.get_eat_turn():
+            player_turn(click_piece, can_piece_eat)
+    elif not turn.get_eat_turn():
+        board.clean()
+
+    return jsonify(output_json)
+
+
+def moving_turn(chosen_piece, click_piece):
```

Comment:
you are moving one piece to a destination, moving is always diagonal so if the source is higher than the destination by 1 then you know you are eating a piece